test/Tizen.NUI.Scene3D.Sample/Scene3DSample.cs

```diff
@@ -479,7 +486,7 @@ void OnKeyEvent(object source, Window.KeyEventArgs e)
         }
         else if (e.Key.State == Key.StateType.Up)
         {
-            if (mModelAnimation?.State == Animation.States.Stopped)
+            if (mModelAnimation != null && mModelAnimation?.State == Animation.States.Stopped)
             {
                 if (mMotionAnimation != null)
```

Comment:
This can be removed.

Reply:
Thanks! I remove '?' keyword. Actually NUI.BaseHandle override the == operator. And that operation is stronger than simple ? operation. So I choose it.

Reply:
Sorry my comment was wrong T.T
I didn't identify mModelAnimation and mMotionAnimation... T.T